sdk/resourcemanager/Azure.ResourceManager/src/autorest.md

```diff
@@ -60,4 +60,8 @@ directive:
     where: $.definitions.*.properties[?(@.enum)]
     transform: >
       $["x-accessibility"] = "public"
+  - from: types.json
```

Comment:
not sure we want to make this change lets check with arm team first since this is how its defined in the swagger